contracts/stellar-interchain-token-service/src/contract.rs

```diff
@@ -311,6 +311,17 @@ impl InterchainTokenServiceInterface for InterchainTokenService {
 
         Ok(())
     }
+
+    #[only_owner]
```

Comment:
If we saved the upgrader in ITS, and used the stored version, technically we could even make this permissionless. There's not much benefit in this situation, so it's fine as-is, but worth considering in the future